### PR TITLE
Lint project code with flake8 and black

### LINKED ISSUE #1096: Check the project with a linter
### Idea summary

Fix all warnings in the project with the wemake-python-styleguide linter.

### Further details

I noticed that the project has a lot of inconsistencies with the generally accepted rules of code design. Can I use the **wemake-python-styleguide** style guide to fix problems and warnings in the project?

### PR DESCRIPTION
**Pull request by brylie**
Closes #1096 
  
## Description

Now that we have flake8 linting our pull requests, we need to run the linter on all files in order to avoid unrelated errors when reviewing new pull requests. This pull request lints all files with `flake8` and `black` formatter.

### CODE REVIEW

File: project/accounts/api.py [hunk 1]
```diff
@@ -304,8 +304,10 @@ def upload_profile_image(request):
                 return JsonResponse(response, status=200)
 
             except Profile.DoesNotExist:
-                response = {"message": f"Profile with user {request.user.username} does not exist",
-                            "error": "ACCOUNT_ERROR"}
+                response = {
```
Comment (AbhijithGanesh) [inline project/accounts/api.py:307]: is the new-line required?

File: project/accounts/api.py [hunk 2]
```diff
@@ -331,7 +333,9 @@ def clear_profile_image(request):
 
             return HttpResponse("Image Deleted")
         except Profile.DoesNotExist:
-            return HttpResponseServerError(reason=f"Profile with id:{request.user.username} does not exist")
+            return HttpResponseServerError(
```
Comment (AbhijithGanesh) [inline project/accounts/api.py:336]: Can this newline be removed ?

File: project/accounts/authentication.py [hunk 3]
```diff
@@ -14,10 +14,12 @@ class ProfileActivationTokenGenerator(PasswordResetTokenGenerator):
     key_salt = "django.contrib.auth.tokens.PasswordResetTokenGenerator"
 
     def _make_token_with_timestamp(self, user, timestamp, legacy=False):
-        """ Token function pulled from Django 1.11 """
+        """Token function pulled from Django 1.11"""
         ts_b36 = int_to_base36(timestamp)
 
-        hash_string = salted_hmac(self.key_salt, str(user.pk) + str(timestamp)).hexdigest()[::2]
+        hash_string = salted_hmac(
```
Comment (AbhijithGanesh) [inline project/accounts/authentication.py:20]: new line again

File: project/accounts/tests/test_models.py [hunk 4]
```diff
@@ -74,16 +85,24 @@ def test_profile_card_summarize(self):
             "request_profile": self.test_profile.first_name,
         }
         self.assertEqual(
-            Profile.objects.card_summarize(self.test_profile, Profile.objects.get(user=self.test_profile.user)), data)
+            Profile.objects.card_summarize(
```
Comment (AbhijithGanesh) [inline project/accounts/tests/test_models.py:88]: New line

**Review by AbhijithGanesh — COMMENTED** [review]
All PEP8 changes are valid

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]
Multiple E501 errors reflect because of the yaml config. Can it be added to the ignore in workflow yml

File: project/accounts/api.py (hunk 1 above)
Comment (brylie) [reply project/accounts/api.py:307]: It was added when formatting the code with black.

File: project/accounts/api.py (hunk 2 above)
Comment (brylie) [reply project/accounts/api.py:336]: We don't want to fight with the code formatter.

File: project/accounts/authentication.py (hunk 3 above)
Comment (brylie) [reply project/accounts/authentication.py:20]: It is alright to add newlines in the code to break up long lines. This is done automatically by black.

File: project/accounts/tests/test_models.py (hunk 4 above)
Comment (brylie) [reply project/accounts/tests/test_models.py:88]: New lines are OK.

**Comment by brylie** [comment]
Yes, E503 is deprecated now so we should tell flake8 to ignore it.

**Comment by AbhijithGanesh** [comment]
@brylie please do, I don't think we need a PR for the same

**Review by AbhijithGanesh — APPROVED** [review]

**Review by gorkemarslan — APPROVED** [review]

**Comment by gorkemarslan** [comment]
@brylie, now no linting issue exists.

**Pull request merged**